docs/UserGuide.md

```diff
@@ -101,11 +102,15 @@ Format: `delete INDEX`
 Example:
 * `delete 4` deletes the 4th client listed in Realodex, provided he/she exists.
 
+Errors:
```

Comment:
Perhaps you can condense this part just to mention that should you key in an invalid input, the error message will describe exactly what is wrong and inform you how to key in the correct input.
This is for consistency with the other commands, especially the Add command where there are too many error messages to list exhaustively.